examples/campaign_management/add_campaign_labels.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example adds a campaign label to a list of campaigns.
+
+This example assumes that a label has already been prepared.
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+_DEFAULT_PAGE_SIZE = 1000
+
+
+def main(client, customer_id, label_id, campaign_ids, page_size):
+    """This code example adds a campaign label to a list of campaigns.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: A client customer ID str.
+        label_id: A label ID to use when searching for campaigns.
+        campaign_ids: A list of campaign IDs to which the label will be added.
+        page_size: An int of the number of results to include in each page of
+            results.
+    """
+
+    # Get an instance of CampaignLabelService client.
+    campaign_label_service = client.get_service(
+        'CampaignLabelService', version='v3')
+
+    # Set the resource name of the label to be added across the campaigns.
+    label_resource_name = f'customers/{customer_id}/labels/{label_id}'
```

Comment:
Use the service's path to create the resource name so it is encapsulated in one place in case the formats change in the future. See for example https://github.com/googleads/google-ads-python/blob/2e11de5f0edbb561b6c186aeef95a5605336723a/examples/basic_operations/add_responsive_search_ad.py#L38

Reply:
Thanks for this example; I was looking for something similar.  Fixed.